### DISCUSSION TITLE
Hacktoberfest is upon us, worth labelling some issues accordingly?

### DISCUSSION THREAD

**Discussion by symonk** [pytest-dev/pytest#7791] [General] [answered]
Would there be any use in taking some low hanging fruit / issues and applying the 'Hacktoberfest' label to them?  Could be a useful tool to bringing new contributors into the project.  

https://hacktoberfest.digitalocean.com/details#maintainers

**Comment by nicoddemus** [comment]
Absolutely! We did that in past Hacktoberfests already.

Which issues should we mark for Hacktoberfest? How about all "easy" and "docs" also receive that label?

**Comment by symonk** [comment, marked as answer]
@nicoddemus sounds good, ok if I go ahead and do that?

**Reply by nicoddemus** [reply]
Sure go ahead, thanks!

**Comment by symonk** [comment]
@nicoddemus seems 2020 is slightly different, we should add a hacktoberfest topic and add hacktoberfest-accepted labels to PRs, please see: https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Reply by symonk** [reply]
looks like we already have the topic :)

**Reply by The-Compiler** [reply]
I don't see why we'd need to add the label, as PRs which are marked as accepted and/or merged count as well.

**Reply by nicoddemus** [reply]
I agree, no need for the label on the PRs, just approving/merging is sufficient. 👍
